extras/mmu.py

```diff
@@ -3366,7 +3366,7 @@ def _form_tip_standalone(self, extruder_stepper_only=False):
                 # Not synchronous gear/extruder movement
                 return (delta > self.encoder_min or filament_present == 1), park_pos
         finally:
-            self._sync_gear_to_extruder(False)
+            self._sync_gear_to_extruder(self.sync_to_extruder, servo=True, in_print=self._is_in_print() or self._is_in_pause())
```

Comment:
@moggieuk for my own knowledge, is there any reason `in_print` is specified as an argument? Could `_sync_gear_to_extruder` use `self._is_in_print() or self._is_in_pause()` so we don't have to track that at the call site?

Reply:
Worth noting, this could also be solved by reverting sync state in the same block that does the early return when changing tools, but this seemed as good a place to put it as any

Reply:
All the sync logic is new so yes this is quite probably a bug.   Not sure if this is the case here but there were use cases for forcing "in_print" is necessary (either real or testing purposes).  But clearing sync_gear_to_extruder isn't correct, it should restore the previous state.  May be easiest/safest to get that state just before changing the sync for tip forming and then just reset it in the finally block